README.md

```diff
@@ -99,6 +99,68 @@ To run all tests, run the following command:
 forge test
 ```
 
+## Deployments
+All commands to deploy: 
+  - Either the full system or just the controller
+  - To mainnet or base 
+  - For staging or production
+
+Can be found in the Makefile, with the nomenclature `make deploy-<domain>-<env>-<type>`.
+
+Deploy a full ALM system to base staging: `make deploy-base-staging-full`
```

Comment:
Staging full deploy does both mainnet and base at the same time and is `deploy-staging-full`